website/data/community.json

```diff
@@ -17,7 +17,7 @@
     "type": "examples-and-training",
     "badges": [
     ],
-    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p>"
+    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p><p>FCFP is priced at $299 but free seats are available to FINOS members.</p>"
```

Comment:
This should not be FCFP right - LFD237? Or should be just say "this course is priced..."